src/Ormolu/Printer/Meat/Declaration/Warning.hs

```diff
@@ -0,0 +1,73 @@
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Ormolu.Printer.Meat.Declaration.Warning
+  ( p_warnDecls
+  , p_warningTxt
+  , WarningType(..)
```

Comment:
I think we should not leak this out of this module. Factually the outside world is interested in two functions: 1) `p_warnDecls` 2) `p_warningTxt ModuleText`. That's exactly what should be exported from this module.

I'll adjust it a bit later.